tf-modules/distribution/main.tf

```diff
@@ -205,3 +212,44 @@ resource "aws_api_gateway_deployment" "s3_credentials" {
   rest_api_id = module.thin_egress_app.rest_api.id
   stage_name  = var.api_gateway_stage
 }
+
+# Egress Api Gateway Log Group Filter
+resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscription_filter" {
+  count           = var.log_destination_arn != null && var.log_api_gateway_to_cloudwatch ? 1 : 0
+  name            = "EgressApiGatewayCloudWatchLogSubscriptionToSharedDestination"
+  destination_arn = var.log_destination_arn
+  filter_pattern  = ""
+  log_group_name  = module.thin_egress_app.egress_log_group
+}
+
+# Egress Lambda Log Group
+resource "aws_cloudwatch_log_group" "egress_lambda_log_group" {
+  count             = var.log_destination_arn != null ? 1 : 0
```

Comment:
It's a style thing, and I realize we're still coming up with best practices for TF, but I usually say `var.log_destination_arn == null ? 0 : 1`. I'll defer to you, though.

Reply:
To be clear, it doesn't really matter, I just need to read a little more carefully when we're using `!= null` in some places and `== null` in others.